### PR TITLE
Up 89

### PR DESCRIPTION
**Pull request by fontainesw**
Organizations.

### CODE REVIEW

File: packages/ui-bindings/Organization/organizationInterface.tsx
Comment (Lou3797) [inline packages/ui-bindings/Organization/organizationInterface.tsx]: For consistency with the rest of the project shouldn't this just be "Organization"?

File: packages/app/pages/Api/agent.tsx
Comment (ConnorHibbs) [inline packages/app/pages/Api/agent.tsx]: Same comment as UP-94. I would move functions that are used by multiple files to a utility directory, or somewhere it is clear they don't belong to any one page. If everything requires from Agent, there is a lot of unnecessary dependency.

File: packages/app/pages/Api/organization.tsx
Comment (ConnorHibbs) [inline packages/app/pages/Api/organization.tsx:40]: [Duplicate of UP-94] I would remove the ternary operator from here. This isn't what the ternary operator was made to do, and I think it leads to unnecessary code complexity. Consider an if-elseif-else block instead

File: packages/app/pages/Api/organization.tsx
Comment (ConnorHibbs) [inline packages/app/pages/Api/organization.tsx]: Same as above, I would remove the ternary operator

File: packages/app/pages/Api/organization.tsx
Comment (ConnorHibbs) [inline packages/app/pages/Api/organization.tsx]: Same as above, I would remove the ternary operator.

File: packages/ui-bindings/Organization/getOrganizationById.tsx
Comment (ConnorHibbs) [inline packages/ui-bindings/Organization/getOrganizationById.tsx:18]: $OrganizationId could lead to confusion, since variables are typically made camelCase and interfaces / types / classes are made UpperCase. I'll still approve it if you feel otherwise, but consider making variables camelCase

**Review by ConnorHibbs — CHANGES_REQUESTED** [review]
Please fix the ternary operators, and look at the other comments. I'm going to run the code locally as well in a second.

File: packages/app/pages/Api/agent.tsx
Comment (fontainesw) [reply packages/app/pages/Api/agent.tsx]: will sync dev when up 94-is merged

File: packages/ui-bindings/Organization/getOrganizationById.tsx
Comment (fontainesw) [reply packages/ui-bindings/Organization/getOrganizationById.tsx:18]: I made this decision to keep form confusing it from props.organazationId. Do you think that is a valid reason to not camelCase it?

File: packages/app/pages/Api/organization.tsx
Comment (fontainesw) [reply packages/app/pages/Api/organization.tsx:40]: there is no ternary operator here.

File: packages/app/pages/Api/organization.tsx
Comment (fontainesw) [reply packages/app/pages/Api/organization.tsx]: resolved

File: packages/app/pages/Api/organization.tsx
Comment (fontainesw) [reply packages/app/pages/Api/organization.tsx]: resolved

**Review by Lou3797 — CHANGES_REQUESTED** [review]

File: packages/ui-bindings/Organization/getOrganizationById.tsx
Comment (ConnorHibbs) [reply packages/ui-bindings/Organization/getOrganizationById.tsx:18]: Personally, I would say since they represent the same information they should have the same name. I'll leave it to your discretion though if you want to change it or not

**Review by ConnorHibbs — APPROVED** [review]
Thanks for fixing the ternary operators!

**Review by Lou3797 — APPROVED** [review]

**Pull request merged**
